validation.js

```diff
@@ -31,5 +35,26 @@ const loginValidation = (data) => {
   return schema.validate(data);
 };
 
+
+// Login validation 
+const validateLogin = (username, password) => {
+  // Define rules
+  const usernameRegex = /^[a-zA-Z0-9_]{6,20}$/;       // Username must be between 6 to 20 characters long and can include letters, numbers, and underscores
```

Comment:
questo puoi esprimerlo anche con funzioni di joi (vedi il mio validation della registrazione)

Reply:
(anche se ho notato ora che hai fatto un po' diversamente rispetto a me quindi potrebbe essere l'unica soluzione la tua, è da vedere)

Reply:
Era fattibile anche con le joi, ma volevo provare ad implementarlo da me. Se nella fase di testing dovessimo riscontrare problemi, lo implemento con le funzioni di joi

Reply:
ok. quando hai modificato tutto e messo a posto fai una push in questa branch e fammi una richiesta di re-review.